src/data/reducers/assets.js

```diff
@@ -23,6 +23,13 @@ const list = (state = [], action) => {
 
 const status = (state = {}, action) => {
   switch (action.type) {
+    case assetActions.CLEAR_ASSETS_STATUS:
+      return {};
+    case assetActions.DELETE_ASSET_SUCCESS:
+      return {
+        response: action.response,
+        text: action.text,
```

Comment:
Recap of our in-person conversation:
* we'll use the existing `type` instead of text to compare against
* the `type` constants will be moved into a separate file

Thanks for the discussion!